src/themes/light-theme/color-component-tokens.js

```diff
@@ -181,6 +181,8 @@ export default (
   inputEnhancerFill: themePrimitives.mono300,
   inputEnhancerFillDisabled: themePrimitives.mono300,
   inputEnhancerTextDisabled: themePrimitives.mono600,
+  inputPlaceholder: themePrimitives.mono800,
+  inputPlaceholderDisabled: themePrimitives.mono800,
```

Comment:
I think these two values should nor be the same. 
Placeholder in a disabled state has `#AFAFAF` color, and  `#757575` when an input is enabled.

Reply:
I changed the disabled state colors to the same values as the `inputTextDisabled` for the light and dark themes.
Also, it looks like for the light theme, the placeholder color is `#545454` not `#757575`

Reply:
The values in my previous comment are from our latest and the current design ^